frontend/public/components/dashboard/top-consumers-card/utils.tsx

```diff
@@ -0,0 +1,87 @@
+import { get } from 'lodash';
+
+const BYTE_UNITS_CONFIG: ByteUnitsConfig = {
+  conversion: 1024,
+  units: {
+    B: 0,
+    Ki: 1,
+    Mi: 2,
+    Gi: 3,
+    Ti: 4,
+    Pi: 5,
+  },
+};
+
+const formatBytes = (
+  bytes: number,
+  unit = null,
+  fixed: number = 2,
+  conversionConfig: ByteUnitsConfig = BYTE_UNITS_CONFIG
+): FormattedByte => {
+  const { units, conversion } = conversionConfig;
+  const unitKeys = Object.keys(units);
+  unit = unit || unitKeys.find(key => bytes < conversion ** (units[key] + 1)) || unitKeys[unitKeys.length - 1];
+  return { value: Number((bytes / conversion ** units[unit]).toFixed(fixed)), unit };
+};
+
+export const formatBytesWithUnits = (bytes: number): string => {
+  const formattedBytes: FormattedByte = formatBytes(bytes);
+  return `${formattedBytes.value} ${formattedBytes.unit}`;
+};
+
+export const formatCpu = (cpuTime: string, fixed: number = 0): string => Number(cpuTime).toFixed(fixed);
```

Comment:
console naming conventions are

```suggestion
export const formatCPU = (cpuTime: string, fixed: number = 0): string => Number(cpuTime).toFixed(fixed);
```